scripts/sommelier

```diff
@@ -183,6 +183,25 @@ start_app() {
     . "$SNAP/sommelier/hooks/pre-start"
   fi
 
+  if [[ -e "${ACTION_ARGS}" ]]; then
+    # Convert Unix paths to Windows paths.
+    for i in "${ACTION_ARGS[@]}"; do
+      if [[ -e "${i}" ]]; then 
+      args+=( "$(winepath -w "$i")" )
+      fi
+    done
+    ACTION_ARGS=( "${args[@]}" )
+  elif [[ "$ACTION_ARGS" == "/add" ]] && [[ "$(basename $RUN_EXE)" == "foobar2000.exe" ]]; then
+    # Convert Unix paths to Windows paths.
+    # Skips first arg /add in loop.
+    for i in "${ACTION_ARGS[@]:1}"; do
+      if [[ -e "${i}" ]]; then
+      args+=( "$(winepath -w "$i")" )
+      fi
+    done
+    ACTION_ARGS=( "/add" "${args[@]}" )
+  fi
+
```

Comment:
I was thinking more of something like this; I think this should work for all apps, including foobar2000. Is my thinking correct?

```suggestion
  # Convert all arguments which point to existing files/dirs to Windows paths.
  for i in "${ACTION_ARGS[@]}"; do
    if [[ -e "${i}" ]]; then
      args+=( "$(winepath -w "$i")" )
    else
      args+=( "$i" )
    fi
  done
  ACTION_ARGS=( "${args[@]}" )
```